contributing.md

```diff
@@ -7,21 +7,14 @@
 1. Add your name on the google doc [bit.ly/codebuddies-hangouts-platform-v2-googledoc](http://bit.ly/codebuddies-hangouts-platform-v2-googledoc) if your name is not listed, and you want to be recognized as a collaborator.
 2. Say hello on the [#codebuddies-meta channel in the Slack](https://codebuddiesmeet.slack.com/messages/codebuddies-meta/). Feel free to ask questions here, or to ask for someone to review your pull request.
 3. [Install Meteor](https://www.meteor.com/install) and Node: [docs.npmjs.com/cli/install](https://docs.npmjs.com/cli/install) if you don't already have them installed.
-4. Please star this repository! We need to reach 100 stars so that we can apply to the [Open Collective](https://opencollective.com/opensource/apply).
+4. Please star this repository! We need to reach 100 stars so that we can apply to the [Open Collective](https://opencollective.com/opensource/apply). [Edit - We're already there!]
 5. Fork this repository! Once you have a copy of this repo on your own account, clone this repo to your computer by typing in something like:
   `git clone https://github.com/codebuddiesdotorg/cb-v2-scratch.git`
 (Replace the URL with your own repository URL path.)
-6. Setup this repository as an upstream branch using:
```

Comment:
Looks generally great, @abhiii5459, except this bit overrides an earlier merge of a PR that edited Step #6, with the instructions about how to set up your forked branch to have an upstream pointing back to the original CB repo. :)

Can you check if you've got it set up? Then you should be able to `git fetch upstream` `git checkout master` `git merge upstream/master` to merge in the latest changes from master.

Thanks!